assets/js/steps/components/num-pad.js

```diff
@@ -14,21 +17,29 @@ import PropTypes from 'prop-types';
  * @return {*}
  * @constructor
  */
-function NumPad({ handleNumpadPress }) {
-    const buttons = ['1', '2', '3', '4', '5', '6', '7', '8', '9', '#', '0', 'C'];
+function NumPad({ handleNumpadPress, okButtonLabel, deleteButtonLabel }) {
+    const buttons = [{ label: '1' }, { label: '2' }, { label: '3' }, { label: '4' }, { label: '5' }, { label: '6' }, { label: '7' }, { label: '8' }, { label: '9' }, { label: deleteButtonLabel || 'Slet', icon: faBackspace, extraClass: 'red' }, { label: '0' }, { label: okButtonLabel || 'Ok', icon: faArrowAltCircleRight, extraClass: 'green' }];
+
     return (
         <div className='num-pad'>
```

Comment:
<img width="586" alt="image" src="https://user-images.githubusercontent.com/15377965/93807110-75a8d600-fc4a-11ea-94d8-bf6bff3c3a13.png">

I changed the design a bit from figma, can be changed back though.